guides/common/modules/con_introduction-to-organization-and-location-context.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Introduction_to_Organization_and_Location_Context_{context}"]
+= Introduction to Organization and Location Context in {Project}
+
+Context in {Project} consists of xref:Managing_Organizations_{context}[Organizations] and xref:Managing_Locations_{context}[Locations].
+
+Most entities in {Project} are associated with at least one organization and location context.
+Since entities and users can generally only access resources and other entities within their own context, organizations and locations form an integral part of {Project}'s user management.
+For example, a user belonging to the _internal_ location is unable to access a DNS service that is exclusively assigned to the _DMZ_ location.
```

Comment:
```suggestion
For example, a user belonging to the _internal_ location is unable to access a DNS that is exclusively assigned to the _DMZ_ location.
```

This use of service is a bitter pill for me to swallow because it means relatively nothing. I think that every time we call something a service, we might need to slap ourselves in the wrist :o)

Reply:
Removed the whole example in favor of Ondrej's.